readme.md

```diff
@@ -42,6 +45,70 @@ print(LaunchAtLogin.isEnabled)
 //=> true
 ```
 
+#### As singleton:
+
+```swift
+import LaunchAtLogin
+
+print(LaunchAtLogin.shared.isEnabled)
+//=> false
+
+LaunchAtLogin.shared.isEnabled = true
+
+print(LaunchAtLogin.shared.isEnabled)
+//=> true
+```
+
+#### Storyboards:
+
+Bind control to  `LaunchAtLogin.isEnabled` ViewController's property:
+
+```
+import Cocoa
+import LaunchAtLogin
+
+class ViewController: NSViewController {
+    @objc dynamic var launchAtLogin = LaunchAtLogin.shared
+}
+```
+![storyboard_binding](.github/storyboard_binding.png)
+
+#### SwiftUI:
+
```

Comment:
While it's useful to document how to use this manually in SwiftUI, I think we could make it even nicer by also including a `Toggle` wrapper:

```swift
LaunchAtLogin.Toggle("Launch at login")
```

Where, like `Toggle`, it would either accept a string title or a label view (see the `Toggle` type definition). And the string label could default to `Launch at login` for even nicer usage.